### DISCUSSION TITLE
Anchor 容器模式，滚动到顶部，第一个link也没有 激活

### DISCUSSION THREAD

**Discussion by Alixhan** [element-plus/element-plus#17977] [Q&A]
我不太清楚这个是故意为之，还是一个bug，滚动到顶部后，没有任何link被激活

**Comment by Fuphoenixes** [comment]
设计如此

**Comment by Fuphoenixes** [comment]
参考[vue官网](https://cn.vuejs.org/guide/introduction.html) 右侧的锚点也是这样设计的

**Comment by kcmeven** [comment]
那是因为最顶上有个标题和描述， 但是其他人用的时候不一定有啊 ， 这里起码给个配置吧

**Comment by zengkaii** [comment]
![image](https://github.com/user-attachments/assets/aae8fca3-dd48-4390-9099-9e684ebf908b)

所以手动处理一下

**Reply by zhangXiaolizi** [reply]
初始手动处理是没问题了，但是每次滚动到顶部，又消失了

**Reply by zengkaii** [reply]
> 初始手动处理是没问题了，但是每次滚动到顶部，又消失了

那应该是你的上间距有问题
